assets/js/app.js

```diff
@@ -258,7 +258,7 @@ let upArrow = $(`
 //function for the "Scroll To Top" button to detect the footer
 $(document).ready(function () {
   $(window).scroll(function () {
-    console.log($(window).scrollTop());
+    // console.log($(window).scrollTop());
```

Comment:
Why is this changed?

Reply:
I was checking my js function is getting executed or not so I used console and commented it for removing the confusion and forgot to uncomment it.